### PR TITLE
Adding vuid 02760 02761 02762, Validate subgroup size control 

### PR DESCRIPTION
**Pull request by ziga-lunarg**
requiredSubgroupSize must be a power of 2 integer and within minSubgroupSize and maxSubgroupSize

### CODE REVIEW

File: layers/parameter_validation_utils.cpp [hunk 1]
```diff
@@ -1591,6 +1599,30 @@ bool StatelessValidation::ValidatePipelineShaderStageCreateInfo(const char *func
                 "VkPipelineShaderStageRequiredSubgroupSizeCreateInfoEXT is included in the pNext chain.",
                 func_name, msg, pCreateInfo->flags);
         }
+        // Check for power of 2
+        if ((required_subgroup_size_features->requiredSubgroupSize & (required_subgroup_size_features->requiredSubgroupSize - 1)) !=
```
Comment (jeremyg-lunarg) [inline layers/parameter_validation_utils.cpp:1603]: It looks like we have an IsPowerOfTwo() function for this type of check.

**Review by jeremyg-lunarg — COMMENTED** [review]
Needs to use IsPowerOfTwo(), otherwise looks good.

File: layers/parameter_validation_utils.cpp (hunk 1 above)
Comment (ziga-lunarg) [reply layers/parameter_validation_utils.cpp:1603]: I forgot about that, thanks!

**Review by jeremyg-lunarg — APPROVED** [review]

**Comment by spencer-lunarg** [comment]
These VUIDs are already covered now

**Pull request closed without merging**